str/helper/mt_extractor.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-function-docstring,no-member
+"""
+This Hail Query script outputs a Hail matrix table  from a BGZIPPED VCF file.
+ analysis-runner --dataset "bioheart" \
+    --description "mt_extractor" \
+    --access-level "test" \
+    --output-dir "str/polymorphic_run/mt/v1" \
+    mt_extractor.py --file-path=gs://cpg-bioheart-test/str/polymorphic_run/bzip/v1/tester_file.vcf.bgz
+
+"""
+
+import hail as hl
+import click
+
+from cpg_utils.hail_batch import output_path, init_batch
+
+
+@click.option(
+    '--file-path',
+    help='GCS file path to BGZIP VCF file.',
+    type=str,
```

Comment:
Not important, but the default type of a click arg is str, so this shouldn't need clarification

Reply:
removed